### ISSUE TITLE
Create a c302 DataTranslator and DataSources

### ISSUE THREAD

**Issue by mwatts15** [openworm/c302#12]
PyOpenWorm has a notion of DataTranslators that describe how data get from
one form to another and each of those forms is called a DataSource. A
DataSource should be created to describe the input to c302 and a DataSource
should be created to describe the output NeuroML files. The DataTranslator
should have a `translate` method which runs c302 with the first DataSource
type and outputs the second. The first DataSource should have a Translation
linking it back to the PyOpenWorm insert_worm DataSource (TBD:
openworm/PyOpenWorm#364).

blocked by openworm/PyOpenWorm#364

**Comment by pgleeson** [comment]
No longer relevant due to move to https://github.com/openworm/ConnectomeToolbox for connectomes info
